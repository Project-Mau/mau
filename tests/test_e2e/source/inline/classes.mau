This paragraph uses [class]("classes", class1, class2).
