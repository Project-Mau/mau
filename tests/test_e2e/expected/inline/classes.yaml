data:
  args: []
  content:
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'This paragraph uses '
      - data:
          args: []
          classes:
          - class1
          - class2
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: classes
          kwargs: {}
          subtype: null
          tags: []
          type: macro.class
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: .
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  kwargs: {}
  subtype: null
  tags: []
  type: document

